test/conformance/conformance_test.go

```diff
@@ -24,18 +26,25 @@ func TestGatewayAPIConformance(t *testing.T) {
 
 	require.NoError(t, v1alpha2.AddToScheme(client.Scheme()))
 
+	setUniquePorts := env.Lookup("CONFORMANCE_UNIQUE_PORTS", "true")
```

Comment:
would `flag.Parse()` be a cleaner solution here ?

```
useUniquePorts = flag.Bool("use-unique-ports", true, "Whether to cleanup base test resources after the run")
func TestMain(m *testing.M) {
    flag.Parse()
    os.Exit(m.Run())
}

TestGatewayAPIConformance(t *testing.T) {
 if *useUniquePorts {
  ....
 } else {
 ....
 }
....
}
```

it would need us to create another make target similar to https://github.com/envoyproxy/gateway/blob/fcdb9b168c823d2fd47dd94c3edfc4c5db984f42/tools/make/kube.mk#L96 that gets called in CI
on the other hand the env approach would only require a change here https://github.com/envoyproxy/gateway/blob/fcdb9b168c823d2fd47dd94c3edfc4c5db984f42/.github/workflows/build_and_test.yaml#L52 but we lose out on the discoverability of the flag

Reply:
Thanks for this comment, will update this later:)